configure/recipes/timescale.rb

```diff
@@ -0,0 +1,47 @@
+unless node.attribute?(:ec2)
+  # Copy config files
+  cookbook_file "/etc/postgresql/12/main/pg_hba.conf" do
+    source "pg_hba.conf"
+    owner "root"
+    group "root"
+    mode "0644"
+  end
+  cookbook_file "/etc/postgresql/12/main/postgresql.conf" do
+    source "postgresql.conf"
+    owner "root"
+    group "root"
+    mode "0644"
+  end
+
+  # Tune pgsql for timescale
+  execute "tune-and-restart-pgsql" do
+    command "sudo timescaledb-tune -yes -quiet; sudo service postgresql restart"
+    user "root"
```

Comment:
This is a somewhat dangerous command because you're manipulating the state of the service directly. In most cases in works how you assume, but you run the risk re-activating the service by accident since this command doesn't consider what the intended state of postgres is.

A slightly fix could be:

```
  execute "tune-and-restart-pgsql" do
    command "sudo timescaledb-tune -yes -quiet"
    user "root"
    notifies :restart, "service[pgsql]", :delayed
    only_if node["configure"]["services"]["pgsql"] && node["configure"]["services"]["pgsql"].include? "start"
end